### PR TITLE
Added editable rows story

### PR DESCRIPTION
**Pull request by ahmetkuslular**
I added a row editing example. 😋

### CODE REVIEW

**Comment by jbetancur** [comment]
Very nice 👍

File: stories/DataTable/Basic/Editable.stories.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            column={{ ...column, editing }}
+            col={col}
+            onChange={formOnChange}
+          />
+        );
+      },
+    };
+  });
+
+  const createColumns = () => {
```
Comment (jbetancur) [inline stories/DataTable/Basic/Editable.stories.js:115]: I would recommend memorizing this function with `useCallback` to keep `DataTable` to minimize RDT re-rendering

**Comment by jbetancur** [comment]
This is a really great example. Thank you for this. Just one comment for review

File: stories/DataTable/Basic/Editable.stories.js (hunk 1 above)
Comment (ahmetkuslular) [reply stories/DataTable/Basic/Editable.stories.js]: I made the fix. Thank you for your comments. 🥳

**Pull request merged**